global.json

```diff
@@ -3,13 +3,13 @@
     "allowPrerelease": true
   },
   "tools": {
-    "dotnet": "8.0.201",
+    "dotnet": "9.0.100-preview.6.24328.19",
     "vs": {
       "version": "17.8.0"
     },
     "xcopy-msbuild": "17.8.5"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "8.0.0-beta.24376.1"
+    "Microsoft.DotNet.Arcade.Sdk": "9.0.0-beta.24379.1"
```

Comment:
Have we confirmed with dnceng that this is what they want for our LTS branch?

Reply:
https://github.com/dotnet/dnceng/issues/3713